components/responsive/views/src/viewport-resize.js

```diff
@@ -0,0 +1,62 @@
+import throttle from 'lodash.throttle'
```

Comment:
We were supposed to comment on this.
This code is duplicated and should not. We could move it to @s-ui/js of to whatever other place proposed by our colleagues.